zcash_primitives/src/consensus.rs

```diff
@@ -240,7 +240,7 @@ impl Parameters for TestNetwork {
             NetworkUpgrade::Blossom => Some(BlockHeight(584_000)),
             NetworkUpgrade::Heartwood => Some(BlockHeight(903_800)),
             NetworkUpgrade::Canopy => Some(BlockHeight(1_028_500)),
-            NetworkUpgrade::Nu5 => None,
+            NetworkUpgrade::Nu5 => Some(BlockHeight(1_599_200)),
```

Comment:
I calculate this to be 8:28pm UTC, October 7th.

Reply:
Did we miss this change in the previous release?

Reply:
I've double-checked this calculation.

Reply:
@defuse yes we did.

Reply:
It wasn't technically necessary, as we don't use these parameters in zcashd. But it's good practice to update them in sync.